cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -688,12 +731,12 @@ def test_can_use_one_content_group_in_multiple_problems(self):
 
         expected = self._get_expected_content_group(usage_for_group=[
             {
-                'url': '/container/{}'.format(vertical.location),
-                'label': 'Test Unit 0 / Test Problem 0'
-            },
-            {
                 'url': '/container/{}'.format(vertical1.location),
                 'label': 'Test Unit 1 / Test Problem 1'
+            },
```

Comment:
Order of items matters when we are comparing the lists and group usage is a list. In actual group configuration information, the usage list had the same problem locators but not in same order without my change. So, that's why getting the expected information in that manner.